.circleci/manager-watchdog.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+max_runtime_seconds=$1
+timeout_seconds=$2
+ci_workflow_id=$3
+
```

Comment:
It might be nice to have a comment that describes the high-level intent of this watchdog script.